sdks/python/container/boot.go

```diff
@@ -76,10 +77,8 @@ func main() {
 	flag.Parse()
 
 	if *setupOnly {
-		if err := processArtifactsInSetupOnlyMode(); err != nil {
-			log.Fatalf("Setup unsuccessful with error: %v", err)
-		}
-		return
+		processArtifactsInSetupOnlyMode()
+		os.Exit(0)
```

Comment:
Given the implementation of processArtifactsInSetupOnlyMode() this is a good cleanup/simplification. That function didn't need to return an error if it was simply failing out all the time.